packages/trader/src/App/Components/Layout/Header/signup-button.jsx

```diff
@@ -1,16 +1,17 @@
-import classNames   from 'classnames';
-import PropTypes    from 'prop-types';
-import React        from 'react';
-import { Button }   from 'deriv-components';
-import { localize } from 'App/i18n';
+import classNames     from 'classnames';
+import PropTypes      from 'prop-types';
+import React          from 'react';
+import { Button }     from 'deriv-components';
+import { goToSignUp } from '_common/base/login';
+import { localize }   from 'App/i18n';
 
 const SignupButton = ({ className }) => (
     <Button
         id='dt_signup_button'
         className={classNames(className, 'btn--primary--default')}
         has_effect
         text={localize('Sign up')}
-        onClick={() => { window.open('https://deriv.com'); }}
+        onClick={goToSignUp}
```

Comment:
can we use redirectToSignUp instead to be consistent with redirectToLogin wording?

Reply:
yeah sure, goToSignUp will be redirectToSingup.